subseas_flex_fcst/config-sample.yaml

```diff
@@ -16,6 +16,17 @@ forecast_var_filePattern: CFSv2_SubXPRCP_CCAFCST_var_Apr_*.txt
 obs_filePattern: CFSv2_SubXPRCP_CCAFCST_obs_Apr_*.txt
 hcst_filePattern: CFSv2_SubXPRCP_CCAFCST_xvPr_Apr_*.txt
 variable: Precipitation Anomaly
+
```

Comment:
Would appreciate some help on better naming conventions here; my idea was that `dataLead` represents the data provider's lead representation, while `lead` represents how we use lead time to calculate the target dates